x-pack/plugins/observability/server/lib/rules/slo_burn_rate/lib/build_query.ts

```diff
@@ -197,17 +186,17 @@ export function buildQuery(
           sources: [{ instanceId: { terms: { field: 'slo.instanceId' } } }],
         },
         aggs: {
-          ...buildWindowAggs(startedAt, slo, burnRateWindows),
+          ...buildWindowAggs(dateEnd, slo, burnRateWindows),
```

Comment:
Why is' dateEnd' passed here instead of `dateStart` here?

Reply:
I think it should still be `dateEnd`, because we are looking backwards with the timewindow so dateEnd is the `now` date that we are starting with and dateStart is lookback date `now - window`. Hopefully that makes sense, and pls let me know if I am missing something!

Reply:
I see, it was a bit confusing that previously we passed `startedAt`, and now we pass `dateEnd`, I am also not sure what was the value of `startedAt` parameter previously, but the current code LGTM.

cc @kdelemme